scripts/get-env-root.sh

```diff
@@ -30,6 +30,21 @@ apt-get install -y iverilog gtkwave
 # Nothing needed
 
 # libfpgalink
+
```

Comment:
This script should not be interactive.

Could this be in bootstrap.sh instead? 

Shouldn't http_proxy and https_proxy already be set in your environment? Does the "sudo -E" above fix the problem?